ste/mgr-JobPartMgr.go

```diff
@@ -436,6 +436,7 @@ func (jpm *jobPartMgr) createPipelines(ctx context.Context) {
 	} else if fromTo.From() == common.ELocation.Benchmark() || fromTo.To() == common.ELocation.Benchmark() {
 		userAgent = common.BenchmarkUserAgent
 	}
+	userAgent = common.GetLifecycleMgr().GetEnvironmentVariable(common.EEnvironmentVariable.UserAgentPrefix()) + userAgent
```

Comment:
>userAgent = common.GetLifecycleMgr().GetEnvironmentVariable(common.EEnvironmentVariable.UserAgentPrefix()) + userAgent [](start = 1, length = 118)

You might want to consider adding a space in between these two if one isn't present in the env var-- Otherwise it might come out like "StgExpAzCopy" rather than "StgExp AzCopy..."